topics/metagenomics/tutorials/cami2-assembly/tutorial.md

```diff
@@ -0,0 +1,1085 @@
+
+## Select the challenge to reproduce
+
+In this section we  choose the challenge to reproduce. As the entire challenge is not a one-day task and takes months to complete, we should focus on one challenge out of four - assembly, genome binning, taxonomic binning, or taxonomic profiling. The choice we make depends on our interests and objectives.
+
+
+### Description of CAMI challenges
+
+
+#### Assembly challenge
+
+Assembly is one of the important components of metagenomic analysis. The challenge is called to assess the quality and performance of tools for assembling strain-resolved genomes. 
+
+Three “challenge” datasets were offered: 
+
+
+
+* marine, 
+* strain madness, and 
+* plant associated data. 
+
+Two types of assemblies were assessed: single-sample assembly, i.e. assembly of reads from only one sample at a time, and co-assembly, i.e. assembly of all reads from several samples together. For the sake of comparison, gold standard assemblies, i.e. the correct result which can be used to compare with and benchmark other methods’ results, were created with all regions covered by at least one read and were provided only after the challenges.
+
+Participants provided 155 submissions and 20 assembler versions of 10 assembling tools to be evaluated during this challenge. The different tools performed differently while measuring various metrics. By CAMI2 developers there are provided [plots](https://www.biorxiv.org/content/biorxiv/early/2021/07/12/2021.07.12.451567/F1.large.jpg?width=800&height=600&carousel=1) of benchmarking analyses of different tools depending on different statistics.
+
+![Assembly tools plot from CAMI2](../../images/cami2/assembly-tools-plots.jpg){:width="70%"}
+
+
+
+Overall, GATB[[3]](https://pubmed.ncbi.nlm.nih.gov/24990603/) ranked best on the strain madness data across most metrics (mismatches, misassemblies, strain recall, and strain precision), while HipMer[[4]](https://ieeexplore.ieee.org/abstract/document/8665813) [[5]](https://ieeexplore.ieee.org/abstract/document/7832788) [[6]](https://www.nature.com/articles/s41598-020-67416-5) on the plant-associated data. Compared to the first CAMI challenge A-STAR[ [7]](https://www.a-star.edu.sg/) considerably improved in genome fraction for the strain madness data. The other leader SPAdes [[8]](https://pubmed.ncbi.nlm.nih.gov/22506599/) [[9]](https://genome.cshlp.org/content/27/5/824?ijkey=fef5a6b24fb0024b39509480f85afaa81724d3e8&keytype2=tf_ipsecsha) was not introduced in the CAMI1 challenge [[1]](https://www.nature.com/articles/nmeth.4458) and performed well in CAMI2. For the type of assembly, the different tools performed differently. Single-sample assembly was done well by all assemblers. HipMer [[4]](https://ieeexplore.ieee.org/abstract/document/8665813) [[5]](https://ieeexplore.ieee.org/abstract/document/7832788) [[6]](https://www.nature.com/articles/s41598-020-67416-5) performed better on single samples as well as for pooled samples.
```

Comment:
Here all of these references need to be fixed. For references please use the proper referencing format (bib entries in tutorial.bib, `{% cite referenceName %}` in the text.

`A-STAR[ [7]](https://www.a-star.edu.sg/)` is a bit of an odd reference, is there a paper that could be used here? If it's just a link and not intended to be a reference, you should do something like `the first CAMI challenge [A-STAR](https://...)`, generally it's best to make the link part of the text itself, not separate from the text.